.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ run_save_node_bin: &run_save_node_bin
 build_machine_environment: &build_machine_environment
   # Specify that we want an actual machine (ala Circle 1.0), not a Docker image.
   docker:
-    - image: meteor/circleci:android-30-node-14
+    - image: cimg/android:2021.09.1-node
```

Comment:
This one uses the current LTS version for node, which would be `16.16.0`, so yeah, don't think this one will work.